gonogo-review/kpi-report.md

```
@@ -0,0 +1,282 @@
+
+Responsiveness (screenshots of the web app taken on an iPhone 13):
+<p align="center">
+    <img src="./screenshots/web/dashboard-mobile.jpeg" alt="Responsive dashboard screenshot" width="200"/>
+    <img src="./screenshots/web/property-detail-mobile.jpeg" alt="Responsive property detail view screenshot" width="200"/>
+</p>
+
+Accessibility:
+
+Each front-end application (web and mobile) is designed to be accessible to all users, including those with disabilities. We follow the **Web Content Accessibility Guidelines (WCAG)** to ensure that our app is usable by everyone. This includes:
+- Visual accessibility features
+    - Use of sufficient contrast between text and backgrounds to ensure readability.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)